font/src/ft/fc/mod.rs

```diff
@@ -47,6 +47,11 @@ pub use self::pattern::{Pattern, PatternRef};
 ///
 /// The returned pattern is the result of Pattern::render_prepare.
 pub fn font_match(config: &ConfigRef, pattern: &mut PatternRef) -> Option<Pattern> {
+    if log_enabled!(log::Level::Debug) {
+        debug!("Matching font with pattern:");
+        pattern.print();
+    }
```

Comment:
I found this helpful for debugging, not sure if it would be appropriate to keep

Reply:
I think we should remove this.